packages/openapi-generator/src/generator.ts

```diff
@@ -262,19 +263,30 @@ async function generateService(
  * @internal
  */
 export async function getInputFilePaths(input: string): Promise<string[]> {
-  if ((await lstat(input)).isFile()) {
-    return [input];
+  if (glob.hasMagic(input)) {
+    return new Promise(resolvePromise => {
+      glob(resolve(input), (_error, paths) => {
+        resolvePromise(
+          paths.filter(path =>
+            /(.json|.JSON|.yaml|.YAML|.yml|.YML)$/.test(path)
+          )
+        );
+      });
+    });
   }
 
-  const directoryContents = await readdir(input);
+  if ((await lstat(input)).isDirectory()) {
+    return new Promise(resolvePromise => {
+      glob(
+        resolve(input, './**/*(*.json|*.JSON|*.yaml|*.YAML|*.yml|*.YML)'),
```

Comment:
This notation is weird. Let's use the more simple and commonly used one:
```suggestion
        resolve(input, '**/*.{json,JSON,yaml,YAML,yml,YML}'),
```